09_fusion_simulations/00_process_simulated_fusions.sh

```diff
@@ -80,3 +80,14 @@ zcat out/fusion_locate_out_reg.txt.gz | cut -f2 | grep -f - data/fusion_simulati
 zcat out/fusion_locate_out_reg.txt.gz | cut -f2 | grep -v -f - data/fusion_simulations_cdna.fq | grep "^@"
 
 # ## deduplicate by CB, UMI, pattern and start - here or in R?
+
+
+# alternative combinatorial piece scan
+# shouldn't we add some room to variations to the roi seqs?
+# why is e13_roi not found in simulations?
+# why are e1 nor e14 not overlapping any of the right/left pieces from reference_anchors_regex.fa?
```

Comment:
@gimoro ,  what do you think?

Reply:
I think we should add some room for variations yes!

Reply:
variation where, 3'? how much? what about the other two questions? The regex should be tailored to the experimental/biological expectations

Reply:
thanks @gimoro 

I think you're fixing stuff and incorporating part of this to other branches, can you also PR here/edit here so we can streamline the edits? or how/where would you prefer to harmonize this, e.g. the simulations?

Reply:
I am working on the leukemia_patient branch, I have a PR request open to merge it to the combinatorial_search branch

Reply:
thanks, I'd prefer you update the regexes and simulations to this branch, perhaps with a PR to this branch, so we keep track of provenances